timer.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Exercise 12</title>
+    <link rel="stylesheet" href="time.css" />
+  </head>
+  <body>
+    <div class="box">
+      <h1>Timer</h1>
+      <div class="timer-container">
+        <div class="timer" id="timer">0</div>
+      </div>
+      <button type="button" id="start_button">Start</button>
+      <button type="button" id="pause_button">Pause</button>
+      <button type="button" id="resume_button">Resume</button>
+      <button type="button" id="stop_button">Stop</button>
+      <button type="button" id="count_button">Count</button>
+      <ol id="count_list"></ol>
```

Comment:
https://github.com/PUP-BSIT/exercise-12-norona-marynicole/blob/3051f6314c15b460cb52a19bf10b693926fa97ff/timer.html#L19-L20

- you can remove the count button and list, since the instruction doesn't mention a functionality for listing previous count values or displaying "Count" button that logs values.  It seems you need only Start, Pause, Resume, and Stop buttons and functions. 
See: https://docs.google.com/presentation/d/1gQDuyCoZ4fFZPoFi2AgGCRM6dU7BrXBYdRHydj5BagM/edit#slide=id.g24aaea973bc_0_51